docs/source/upcoming_release_notes/974-Add_binning_and_region_size_control_to_PCDSAreaDetectorTyphos.rst

```diff
@@ -0,0 +1,31 @@
+974 Add binning and region size control to PCDSAreaDetectorTyphos
+#################
+
+API Changes
+-----------
+- N/A
+
+Features
+--------
+- N/A
+
+Device Updates
+--------------
+- N/A
+
+New Devices
+-----------
+- PCDSAreaDetectorTyphos class: Add in signals for camera binning and region
+                                size control.
```

Comment:
```suggestion
- PCDSAreaDetectorTyphos class: Add in signals for camera binning and region size control.
```
Our release notes aren't going to be monospaced, so this will just end up getting spaced funny, unfortunately